### ISSUE TITLE
[ENG-949] Video duration should be displayed in the Media View

### ISSUE THREAD

**Issue by utkubakir** [spacedriveapp/spacedrive#1168]
<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-949](https://linear.app/spacedriveapp/issue/ENG-949/video-duration-should-be-displayed-in-the-media-view)</sub>

**Comment by utkubakir** [comment]
To confirm, we don't have video duration in the db right now right @brxken128 ?

**Comment by brxken128** [comment]
> To confirm, we don't have video duration in the db right now right @brxken128 ?
> 
> 

I don't think so, but I'm also unsure because I don't handle video stuff. It could be attached to `Object` or something, but it's not populated by the metadata crate.

**Comment by brxken128** [comment]
@brxken128 I don't know, I don't deal with `ffmpeg` or videos, anything related to that. You'd be better off asking @ericson.ds999.

I will however be working on gathering video metadata, while making use of our `ffmpeg` crate as 99% of the work is already done.

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
